packages/feedback/src/widget/Actor.ts

```diff
@@ -0,0 +1,57 @@
+import type { FeedbackComponent, FeedbackConfigurationWithDefaults } from '../types';
+import { Icon } from './Icon';
+import { createElement as h } from './util/createElement';
+
+interface Props {
+  options: FeedbackConfigurationWithDefaults;
+  onClick?: (e: MouseEvent) => void;
+}
+
+interface ActorComponent extends FeedbackComponent<HTMLButtonElement> {
+  /**
+   * Shows the actor element
+   */
+  show: () => void;
+  /**
+   * Hides the actor element
+   */
+  hide: () => void;
+}
+
+/**
+ *
+ */
+export function Actor({ options, onClick }: Props): ActorComponent {
+  function _handleClick(e: MouseEvent): void {
+    onClick && onClick(e);
+  }
+
+  const $el = h(
```

Comment:
Also the `$el` convention is not one that is common in the SDK codebase, IMHO we can just use `el` here...?